inference-engine/thirdparty/clDNN/tests/test_cases/resample_gpu_test.cpp

```diff
@@ -777,6 +777,259 @@ INSTANTIATE_TEST_CASE_P(smoke,
+            }
+        }
+
+        return true;
+    }
+
+    cldnn::engine_configuration get_engine_config(bool enable_profiling = false, const std::string& dump_dir = "") {
+        this->enable_profiling = enable_profiling;
+        cldnn::engine_configuration cfg {enable_profiling, false, false, "", "", true, "", dump_dir};
+        return cfg;
+    }
+
+    void execute_compare(const caffe_resample_random_test_params& params, bool check_result) {
+        // auto cfg = get_engine_config(false, "/home/work/cl_dump/");
+        // auto eng = cldnn::engine(cfg););
+        auto eng = cldnn::engine();
+
+        auto in_layout = layout(params.input_type, format::bfyx, params.input_size);
+        auto in_mem = memory::allocate(eng, in_layout);
+        fill_random(in_mem);
+
+        /// bfyx
+        cldnn::topology topo;
+        topo.add(input_layout("in", in_layout));
+        auto prim = resample("resample", "in", params.output_size, params.num_filter, params.operation_type);
+        prim.align_corners = params.align_corners;
+        prim.pads_begin = params.pads_begin;
+        prim.pads_end = params.pads_end;
+        topo.add(prim);
+
+        auto build_opts = build_options();
+        build_opts.set_option(build_option::outputs({"resample"}));
+
+        auto net = network(eng, topo, build_opts);
+        net.set_input_data("in", in_mem);
+
+        auto result = net.execute();
+        auto output = result.at("resample").get_memory();
+
+        std::string kernel = "";
+        for (auto& info : net.get_primitives_info()) {
+            if (info.original_id == "resample")
+                kernel = info.kernel_id;
+        }
+
+        double ref_exec_time = 0;
+        for (auto& pri : net.get_executed_primitives()) {
+            if (pri.first == "resample") {
+                for (auto& pi : pri.second.get_profiling_info()) {
+                    if (pi.name == "executing") {
+                        ref_exec_time = std::chrono::duration_cast<std::chrono::microseconds>(pi.value->value()).count() / 1000.0;
+                    }
+                }
+            }
+        }
+
+        // Execut resample_opt
+        // auto eng_opt = cldnn::engine(cfg);
+        auto eng_opt = cldnn::engine();
+
+        cldnn::topology topo_opt;
+        topo_opt.add(input_layout("in", in_layout));
+        topo_opt.add(reorder("in_to_input_type", "in", params.in_format, params.input_type));
+        auto prim_opt = resample("resample_opt", "in_to_input_type", params.output_size, params.num_filter, params.operation_type);
+        prim_opt.align_corners = params.align_corners;
+        prim_opt.pads_begin = params.pads_begin;
+        prim_opt.pads_end = params.pads_end;
+        topo_opt.add(prim_opt);
+        topo_opt.add(reorder("res_to_bfyx", "resample_opt", format::bfyx, params.input_type));
+
+        auto build_opts_opt = build_options();
+        build_opts_opt.set_option(build_option::outputs({"resample_opt", "res_to_bfyx"}));
```

Comment:
Did you intend to use `resample_opt` through layout of bfyx? why don't you force the implementation?

Reply:
Replaced to force_imple